src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -317,7 +319,9 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			selectedCell = this.codeCells.find(c => c.cellModel.id === this.activeCellId);
 		}
 		selectedCell.toggleEditMode();
-		this.setActiveCellEditActionMode(selectedCell.cellModel.isEditMode);
+		if (this.model.activeCell.cellType !== CellTypes.Code) {
```

Comment:
What's this change for?

Reply:
I think I added this because of a console error. I'll remove it for now